OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -55,45 +78,18 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IChildService, ChildService>();
             services.AddTransient<IWorkshopService, WorkshopService>();
             services.AddTransient<ITeacherService, TeacherService>();
-            services.AddTransient<IOrganizationService, OrganizationService>();
+            services.AddTransient<IProviderService, ProviderService>();
 
             services.AddTransient<IEntityRepository<Child>, EntityRepository<Child>>();
             services.AddTransient<IEntityRepository<Teacher>, EntityRepository<Teacher>>();
             services.AddTransient<IEntityRepository<Workshop>, EntityRepository<Workshop>>();
 
-            services.AddTransient<IOrganizationRepository, OrganizationRepository>();
+            services.AddTransient<IProviderRepository, ProviderRepository>();
 
             // Register the Swagger generator, defining 1 or more Swagger documents
             services.AddSwaggerGen();
 
             services.AddAutoMapper(typeof(Startup));
         }
-
-        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
```

Comment:
why this method removed?

Reply:
It is not removed, i put this method over ConfigureServices, because of the StyleCope